packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -18,7 +19,7 @@ import { Spacer } from '../Spacer/Spacer'
 import { useTheme } from '../../utils'
 
 // TODO: Replace with global setting
-export const SNACKBAR_DEFAULT_OFFSET: number = 50
+export const SNACKBAR_DEFAULT_OFFSET: number = Platform.OS === 'ios' ? 25 : 0
```

Comment:
Will probably change either on the offset follow-up story or the unit tests one, but just noting we have the isIOS util function as a dedicated shorthand.